docs/element-definitions.md

```diff
@@ -520,23 +569,23 @@ Type Element matching only the character string `"rocket science"`.
 
 ### Array Element
 
-Type with domain of all finite lists.
+Type with domain of all finite lists of values.
 
 
 #### Template
 - `element` - `"array"`
 - `attributes`
   - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
-    - `fixed` ([String](#string-element)) - Restricts domain to a positionally typed fixed-length list over types in content. This type class is usually called a <i>tuple</i>, <i>sum type</i> or <i>Π-type</i>.
+    - `fixed` ([String](#string-element)) - Restricts domain to a positionally typed fixed-length list over types in content.  Further applies the `fixed` type attribute on items typing a single value or specifying a default value.
     - `fixedType` ([String](#string-element)) - Restricts domain to a list of types given in `content`.
   - `validation` - _reserved for future use_
-  - `samples` ([Array[Array]](#array-element)) - array of alternative sample values for this Element; type of items in `samples` MUST match the type of this Element
-  - `default` (Array) - default value for this Element; type of `default` MUST match the type of this Element
-- `content` - finite list of Elements
+  - `samples` ([Array[[Array](#array-element)]](#array-element)) - Alternative sample values for this Element; type of items in `samples` MUST match the type this Element describes
```

Comment:
Bad linking. Use something similar to `([Array](#array-element)[[String](#string-element)])`